build.sbt

```diff
@@ -514,6 +515,7 @@ lazy val scaladoc = configureAsSubproject(project)
   .settings(
     name := "scala-compiler-doc",
     description := "Scala Documentation Generator",
+    libraryDependencies += jqueryDep,
```

Comment:
TODO: jQuery file from webjar must be included into scaladoc artifcact, so the binary generated by `dist/mkPack` work.

Reply:
Added in https://github.com/scala/scala/commit/6d0f2be033920a902efc249be860d0e7160acfe2